src/main/java/TestJunitRunner.java

```diff
@@ -0,0 +1,66 @@
+import managers.WebDriverManager;
+
+import org.junit.jupiter.api.*;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import pageobjects.HomePage;
+import pageobjects.RegisterPage;
+
+public class TestJunitRunner {
+    WebDriverManager webDriverManager = new WebDriverManager("Chrome");
+
+    @BeforeEach
+    public void configureTestPreconditions() {
+
+        webDriverManager.getDriver4Web().get("https://demo.opencart.com/");
+
+    }
+
+    @AfterEach
+    public void tearDownActivities() {
+        webDriverManager.getDriver4Web().close();
+        webDriverManager.getDriver4Web().quit();
+    }
+
+
+    @Test
+    @DisplayName("The registration page can be accessed from navigation bar")
+
+    public void registerPageIsOpened() {
+//        WebDriverManager webDriverManager = new WebDriverManager("Chrome");
```

Comment:
you could delete them as part of the next PR as they are not applicable any more